api/main.go

```
@@ -0,0 +1,54 @@
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "server",
+	Short: "Start the API server",
+	Run: func(cmd *cobra.Command, args []string) {
+		port, _ := cmd.Flags().GetString("port")
+
+		if port == "" {
+			fmt.Println("Error: --port flag is required")
+			cmd.Usage()
```

golangci-lint (1.62.2)

Error return value of `cmd.Usage` is not checked

(errcheck)